src/Controls/src/Core/Button/Button.iOS.cs

```diff
@@ -5,18 +5,377 @@
 using Microsoft.Maui.Controls.Platform;
 using Microsoft.Maui.Graphics;
 using Microsoft.Maui.Handlers;
+using UIKit;
+using CoreGraphics;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Maui.Controls
 {
 	public partial class Button
 	{
+		CGSize _originalImageSize = CGSize.Empty;
+
+		protected override Size MeasureOverride(double widthConstraint, double heightConstraint)
```

Comment:
Note: To add MeasureOverride, I needed to add this method to the publicAPI.Unshipped. Do-able for SR?

Reply:
Yes, that's fine